x-pack/plugin/mapper-counted-keyword/src/main/java/org/elasticsearch/xpack/countedkeyword/CountedKeywordFieldMapper.java

```diff
@@ -78,16 +78,24 @@ public class CountedKeywordFieldMapper extends FieldMapper {
     public static final String CONTENT_TYPE = "counted_keyword";
     public static final String COUNT_FIELD_NAME_SUFFIX = "_count";
 
-    public static final FieldType FIELD_TYPE;
+    private static final FieldType FIELD_TYPE_INDEXED;
```

Comment:
Maybe also update the `CountedKeywordFieldMapperTests` test case to have a test similar to `KeywordFieldMapperTests$testDisableIndex()`?

Reply:
Added in https://github.com/elastic/elasticsearch/pull/103646/commits/47732ce0b522f2c6b9332033dc6f905ca478ad06.